src/main/java/org/folio/marc4ld/service/marc2ld/normalization/MarcRecordNormalizer.java

```diff
@@ -0,0 +1,8 @@
+package org.folio.marc4ld.service.marc2ld.normalization;
+
+import org.marc4j.marc.Record;
+
+public interface MarcRecordNormalizer {
```

Comment:
`MarcRecordPunctuationNormalizer` (or `MarcPunctuationNormalizer`) may be a better name for this class?

Reply:
renamed to MarcPunctuationNormalizer.